sdk/python/feast/repo_operations.py

```diff
@@ -104,6 +104,7 @@ def get_repo_files(repo_root: Path) -> List[Path]:
         p.resolve()
         for p in repo_root.glob("**/*.py")
         if p.is_file() and "__init__.py" != p.name
+        and not p.name.endswith(".ipynb") # Ignore Jupyter Notebook files during feast apply
```

Comment:
can we make this a list of files we ignore and add `.ipynb` to it?

Reply:
Interestingly, I see `.ipynb_checkpoints` in the `ignore_paths` but I have experienced this issue locally very frequently.

Reply:
me too, I just used to do `find . -name ".ipynb_checkpoints" -type d -exec rm -rf {} +`

Reply:
yeah i do something similar